locales/en-US/strings.properties

```diff
@@ -4,10 +4,10 @@ default_label_loading=Loading…
 header_top_sites=Top Sites
 header_highlights=Highlights
 # LOCALIZATION NOTE(header_stories): This is followed by (header_stories_by)
-header_stories=Trending Stories
+header_stories=Top Stories
 # LOCALIZATION NOTE(header_stories_by): This is followed by a logo of the
-# corresponding content (trending stories) provider
-header_stories_by=by
+# corresponding content (stories) provider
+header_stories_by=from
```

Comment:
Might as well change the id to `header_stories_from` and the localization note for the previous "followed by" is no longer accurate as these two are separate now.